src/main/webapp/app/app-routing.module.ts

```diff
@@ -5,8 +5,6 @@ import { UserRouteAuthorityAccessService } from 'app/core/auth/user-route-author
 
 import { errorRoute } from './layouts/error/error.route';
 import { navbarRoute } from './layouts/navbar/navbar.route';
-
-import { DEBUG_INFO_ENABLED } from 'app/app.constants';
```

Comment:
rather set the variable value to `false`.

Reply:
Hmm glaube das hat IntelliJ automatisch gelöscht. Es gibt kein DEBUG_INFO_DISABLED

Reply:
yep, im PR von Marc schon abgesprochen. ist ok so.